utils/irqbalance/files/irqbalance.init

```diff
@@ -33,11 +33,22 @@ start_service() {
 	}
 	config_list_foreach irqbalance banirq handle_banirq_value
 
+	config_get debug irqbalance debug 0
+
 	procd_open_instance "irqbalance"
 	if [ -n "$banned_cpulist" ]; then
 		procd_set_param env IRQBALANCE_BANNED_CPULIST="$banned_cpulist"
 	fi
-	procd_set_param command /usr/sbin/irqbalance -f -c "$deepestcache" -t "$interval" "$banirq"
```

Comment:
why do you remove these quotes?
See https://www.shellcheck.net/wiki/SC2086

Reply:
The removal of the quotes fixes https://github.com/Irqbalance/irqbalance/issues/297

Reply:
Oh I see it.

The real problem is `$banirq` gets quoted, then it is evaluated as a single parameter.
For example, `banirq="-i 1 -i 2"`, the expected behavior is that parameter `-i` is passed twice, with values of `1` and `2` respectively.
But due to `$banirq` gets quoted, `-i 1 -i 2` are passed as a single completed parameter.
Guessing irqbalance just ignores unknown parameters.

I think `handle_banirq_value` can call `procd_append_param command -i "$1"` directly.

Reply:
That's an interesting idea, I will test that.